File: gvpr/Dart/GvprLexerBase.dart
import 'package:antlr4/antlr4.dart';
import 'dart:io';
import 'dart:core';
import 'dart:convert';
import 'dart:collection';
import 'gvprLexer.dart';

abstract class GvprLexerBase extends Lexer
{
    GvprLexerBase(CharStream input) : super(input)
    {
    }

    bool IsColumnZero()
    {
        return this.charPositionInLine == 1;
    }
}

File: gvpr/Cpp/GvprLexerBase.cpp
#include "antlr4-runtime.h"
#include "GvprLexerBase.h"
#include "gvprLexer.h"

GvprLexerBase::GvprLexerBase(antlr4::CharStream * input) : antlr4::Lexer(input)
{
    _input = input;
}

bool GvprLexerBase::IsColumnZero()
{
    return this->getCharPositionInLine() == 1;
}
